### PR TITLE
DAOS-9965 test: Update JIRA ID in container/open.py

### PR DESCRIPTION
**Pull request by shimizukko**
Update the JIRA ID.
Also refactor the tags.

Skip-unit-tests: true
Test-tag: container_open
Signed-off-by: Makito Kano <makito.kano@intel.com>

### CODE REVIEW

**Review by dinghwah — DISMISSED** [review]
LGTM

File: src/tests/ftest/container/open.py [hunk 1]
```diff
@@ -34,16 +34,18 @@ class OpenContainerTest(TestWithServers):
     """
 
     def test_container_open(self):
-        """JIRA ID: DAOS-3223
+        """JIRA ID: DAOS-9965
```
Comment (daltonbohning) [inline src/tests/ftest/container/open.py:37]: Does it make sense to keep both for reference/changelog?
```suggestion
        """JIRA ID: DAOS-3223
           JIRA ID: DAOS-9965
```

**Review by daltonbohning — DISMISSED** [review]
LGTM, but minor comment

**Review by daltonbohning — APPROVED** [review]
LGTM

**Review by dinghwah — APPROVED** [review]
LGTM

File: src/tests/ftest/container/open.py (hunk 1 above)
Comment (shimizukko) [reply src/tests/ftest/container/open.py:37]: Done

**Pull request merged**